package.json

```diff
@@ -27,25 +27,24 @@
   },
   "homepage": "https://github.com/NerdWalletOSS/apollo-invalidation-policies#readme",
   "peerDependencies": {
-    "@apollo/client": "^3.3.0"
+    "@apollo/client": "^3.3.7"
   },
   "dependencies": {
     "@types/graphql": "^14.5.0",
-    "@types/lodash": "^4.14.149",
-    "@types/uuid": "^7.0.2",
-    "graphql": "^15.0.0",
-    "lodash": "^4.17.15",
-    "typescript": "^3.8.3",
+    "@types/lodash": "^4.14.168",
```

Comment:
Thanks for bumping the libs. Were you able to do much testing with all these bumps? My concern would be introducing regressions. I can test it out in some libs.

Reply:
The test suites is passing and I'm not experiencing any issues so far. But I can remove the bump if you'd rather be a bit more cautious.

Reply:
It's fine to bump them, the test suite should catch regressions.